index.html

```diff
@@ -90,7 +90,234 @@ <h1 class="primary-text-header">I’m Steven Levoyer to see you!</h1>
+                                <li class="tag"><a href="#" class="text-s">html</a></li>
+                                <li class="tag"><a href="#" class="text-s">css</a></li>
+                                <li class="tag"><a href="#" class="text-s">javaScript</a></li>
+                            </ul>
+                        </div>
+                        <div class="action">
+                            <button type="button" class="action-button">
+                                <p class="text-a">See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-works">
+                    <img class="snapshoot" src="./img/Snapshoot-4.svg">
+                    <div class="left-block">
+                        <div class="primary-text-work">
+                            <h2 class="project-title">Multi-Post Stories</h2>
+                            <div class="frame-2">
+                                <div class="client">
+                                    <p class="text-p">CANOPY</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <p class="text-p">Back End Dev</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year">
+                                    <p class="text-p">2015</p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="secondary-text-work">
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        </div>
+                        <div class="tags">
+                            <ul class="list-tag">
+                                <li class="tag"><a href="#" class="text-s">html</a></li>
+                                <li class="tag"><a href="#" class="text-s">css</a></li>
+                                <li class="tag"><a href="#" class="text-s">javaScript</a></li>
+                            </ul>
+                        </div>
+                        <div class="action">
+                            <button type="button" class="action-button">
+                                <p class="text-a">See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <!--End-Works-->
+        <!--About-me-->
+        <section aria-label="About-me">
+            <div class="base-about-me">
+                <div class="about-my-self">
+                    <div class="top">
+                        <h1 class="primary-text-about-me">About
```

Comment:
- [ ] Kindly use only a single h1 tag in your document. This is because screen readers depend on the h1 tag to know the title of your page. Since this is your portfolio, the h1 can be your name and every other heading should be between h2 to h5. Kindly check [here](https://www.investisdigital.com/blog/seo/using-h1-tag-improves-search-engine-ranking#:~:text=Your%20website%20should%20have%20only,tags%20in%20the%20page%20code).) for more info.